README.md

```
@@ -38,21 +38,30 @@ SB_SERVICE_ROLE_KEY=eyJhb.....
 
 `NEXT_PUBLIC_SUPABASE_ANON_KEY` と `SB_SERVICE_ROLE_KEY` は、[tasq-sb-dev](https://github.com/shiftb-hub/tasq-sb-dev) で `npx supabase start` を実行したときに表示される値を設定してください。
 
-### 4. データベースの初期化
+### 4. テスト用の環境変数の設定ファイル (.env.test) の作成
+
+プロジェクトのルートフォルダに `.env.test` (Vitest実行時に参照する環境変数の設定ファイル) を新規作成し、以下の内容を記述してください。
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)